rbd/pool_nautilus.go

```diff
@@ -122,3 +122,61 @@ func PoolStatsCreate() *PoolStats {
 func (poolStats *PoolStats) Destroy() {
 	C.rbd_pool_stats_destroy(poolStats.stats)
 }
+
+// PoolStatOptions represents a group of configurable pool stat options.
+type PoolStatOptions C.rbd_pool_stat_option_t
+
+const (
+	// PoolStatOptionImages is the representation of
+	// RBD_POOL_STAT_OPTION_IMAGES from librbd.
+	PoolStatOptionImages = PoolStatOptions(C.RBD_POOL_STAT_OPTION_IMAGES)
+	// PoolStatOptionImageProvisionedBytes is the representation of
+	// RBD_POOL_STAT_OPTION_IMAGE_PROVISIONED_BYTES from librbd.
+	PoolStatOptionImageProvisionedBytes = PoolStatOptions(C.RBD_POOL_STAT_OPTION_IMAGE_PROVISIONED_BYTES)
+	// PoolStatOptionImageMaxProvisionedBytes is the representation of
+	// RBD_POOL_STAT_OPTION_IMAGE_MAX_PROVISIONED_BYTES from librbd.
+	PoolStatOptionImageMaxProvisionedBytes = PoolStatOptions(C.RBD_POOL_STAT_OPTION_IMAGE_MAX_PROVISIONED_BYTES)
+	// PoolStatOptionImageSnapshots is the representation of
+	// RBD_POOL_STAT_OPTION_IMAGE_SNAPSHOTS from librbd.
+	PoolStatOptionImageSnapshots = PoolStatOptions(C.RBD_POOL_STAT_OPTION_IMAGE_SNAPSHOTS)
+	// PoolStatOptionTrashImages is the representation of
+	// RBD_POOL_STAT_OPTION_TRASH_IMAGES from librbd.
+	PoolStatOptionTrashImages = PoolStatOptions(C.RBD_POOL_STAT_OPTION_TRASH_IMAGES)
+	// PoolStatOptionTrashProvisionedBytes is the representation of
+	// RBD_POOL_STAT_OPTION_TRASH_PROVISIONED_BYTES from librbd.
+	PoolStatOptionTrashProvisionedBytes = PoolStatOptions(C.RBD_POOL_STAT_OPTION_TRASH_PROVISIONED_BYTES)
+	// PoolStatOptionTrashMaxProvisionedBytes is the representation of
+	// RBD_POOL_STAT_OPTION_TRASH_MAX_PROVISIONED_BYTES from librbd.
+	PoolStatOptionTrashMaxProvisionedBytes = PoolStatOptions(C.RBD_POOL_STAT_OPTION_TRASH_MAX_PROVISIONED_BYTES)
+	// PoolStatOptionTrashSnapshots is the representation of
+	// RBD_POOL_STAT_OPTION_TRASH_SNAPSHOTS from librbd.
+	PoolStatOptionTrashSnapshots = PoolStatOptions(C.RBD_POOL_STAT_OPTION_TRASH_SNAPSHOTS)
+)
+
+// AddOption adds the given PoolStatOptions to PoolStats.
+// Pass the address of a variable where the value of the option needs to be stored.
+//
```

Comment:
Maybe mention that after calling `AddOption()`, calling `Get()` can be used to retrieve the values for the options.

Reply:
Fixed